grakn-graph/src/test/java/ai/grakn/graph/internal/ResourceTest.java

```diff
@@ -133,13 +133,13 @@ public void testToString() {
         assertTrue(concept2.toString().contains("Value"));
     }
 
+    // this is deliberately an incorrect type for the test
+    @SuppressWarnings("unchecked")
     @Test
     public void testInvalidDataType(){
         ResourceType stringResourceType = graknGraph.putResourceType("Strung", ResourceType.DataType.STRING);
         expectedException.expect(RuntimeException.class);
         expectedException.expectMessage(INVALID_DATATYPE.getMessage("1", String.class.getName()));
-        // this is deliberately an incorrect type for the test
-        //noinspection unchecked
```

Comment:
`@SuppressWarnings` is recognised by the Java compiler, whereas the comment is not.